uber-wrapper.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+# This will replace any variable references in these files
+# If you want to add any additional settings here just add
```

Comment:
I'm somewhat late asking this but what does this mean? How would I get the 600+ lines of extra config I have into the container at runtime? I don't see any broad "the rest of the config" variable in the template.

Reply:
Oh, I misunderstood what you wanted. This doesn't currently have a nice way to pull in another file. The right answer there probably depends on your use-case.

If we are deploying this to prod I think the best way is probably going to involve tweaking uber-wrapper.sh to reach out to some service and grab the current config file. This could be a github repo or some database somewhere? @jasonspriggs probably has ideas on where the config could be stored.

For local development I would probably replace uber-development.ini.template with your config file. You can either copy in the variables from the current one, or just ignore them and hardcode things as they currently are.

Otherwise I can look at doing something better, I'm not sure what the best method is here.

Reply:
Yea, grabbing the config file from Somewhere Else sounds good for remote servers. Given that we eventually want to move most of development.ini to the DB, I think just editing uber-development.ini.template for local dev sounds good!

And yea I think I wasn't clear, the env stuff is what MFF uses on remote servers (staging/prod). Running the docker containers with an env file should work there!